Core/src/org/sleuthkit/autopsy/contentviewers/Metadata.java

```diff
@@ -130,11 +136,14 @@ private void addRow(StringBuilder sb, String key, String value) {
         "Metadata.tableRowTitle.imageType=Type",
         "Metadata.tableRowTitle.sectorSize=Sector Size",
         "Metadata.tableRowTitle.timezone=Time Zone",
-        "Metadata.tableRowTitle.deviceId=Device ID"})
+        "Metadata.tableRowTitle.deviceId=Device ID",
+        "Metadata.tableRowTitle.acquisitionDetails=Acquisition Details",
+        "Metadata.nodeText.unknown=Unknown"})
```

Comment:
My feeling is that it's the "UNKNOWN" that is wrong. The only other things in the window that are all caps are acronyms. It would look especially strange with an image data source since that doesn't even display the other "UNKNOWN".
![acqmetadata](https://user-images.githubusercontent.com/3235872/51401242-46a69c80-1b18-11e9-9d30-5d6574a89fe2.png)

Reply:
Okay 👍